apps/api/src/app/workflows-v2/workflow.controller.ts

```diff
@@ -161,6 +164,23 @@ export class WorkflowController {
     );
   }
 
+  @Get('/:workflowId/step/:stepUuid/metadata')
```

Comment:
question: i wonder if we can do a clean route so in the future we will use the same endpoint to get all the step data. wdyt?


```suggestion
  @Get('/:workflowId/step/:stepUuid')
```

Reply:
if it will be in produciton anyhow you won't be able to do it, also the variables will not be in a natural place in a get step endpont, I rather keep it for now and handle it in the future if it will happen